oio/cli/object/object.py

```diff
@@ -389,6 +395,14 @@ def take_action(self, parsed_args):
         if parsed_args.auto:
             container = self.flatns_manager(obj)
         properties = parsed_args.property
+        if parsed_args.tag:
+            from oio.container.lifecycle import TAGGING_KEY
+            tags = '<Tagging><TagSet>'
+            for k, v in parsed_args.tag.iteritems():
+                tags += '<Tag><Key>%s</Key><Value>%s</Value></Tag>' % (k, v)
+            tags += '</TagSet></Tagging>'
```

Comment:
This probably builds a good enough XML document, but unfortunately it will override any previous tagging information. I see two solutions:
- Clearly define this behavior in the help of the command line.
- Use another method to build the XML. For example we can load the existing document, parse it with etree (not in the CLI module), and patch it with the new or deleted tags.